### PR TITLE
fix(core-notification): make error text meet WCAG AAA compliance

### PR DESCRIPTION
**Pull request by marcod1419**

### CODE REVIEW

File: packages/Notification/Notification.jsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ const renderContent = (variant, children) => {
 
   if (variant === 'error') {
     return (
-      <ColoredTextProvider colorClassName={messagingStyles.errorText}>
+      <ColoredTextProvider colorClassName={messagingStyles.errorTextNotification}>
```
Comment (ryanoglesby08) [inline packages/Notification/Notification.jsx]: The `ColoredTextProvider` was invented as a sneaky way of changing text color without changing the public API of the Typography components.

If the Notification is now just going to use the default text color, then there is no more need for the ColoredTextProvider component in this case, and you maybe can just remove its use from Notification altogether.

`color-shark` is just the normal `color-text`. https://github.com/telusdigital/tds/blob/master/packages/colours/_colours.scss#L21

File: packages/Notification/Notification.jsx [hunk 2]
```diff
@@ -30,14 +28,6 @@ const renderIcon = icon => <DecorativeIcon symbol={icon.symbol} variant={icon.co
 const renderContent = (variant, children) => {
```
Comment (ryanoglesby08) [inline packages/Notification/Notification.jsx]: The `content` variable, and actually the entire `renderContent` function, is not necessary anymore. I'd recommend removing it and inlining this into the main component.

File: packages/Notification/__tests__/Notification.spec.jsx [hunk 3]
```diff
@@ -54,11 +53,7 @@ describe('<Notification />', () => {
     it('bolds and colors the content', () => {
```
Comment (ryanoglesby08) [inline packages/Notification/__tests__/Notification.spec.jsx]: This test needs a slight name change, and there is no more color change.

**Comment by ryanoglesby08** [comment]
@marcod1419 I realized something else about this one. If the color contrast here was not up to our standards, how come our e2e accessibility test is passing? Are we not configuring aXe properly to catch this level of color contrast?

It would be great if you could see if you can get a failing accessibility test on this component first, and then fix it with the text color change.

**Comment by marcod1419** [comment]
@ryanoglesby08 Just ran the e2e tests and they're passing. Looks like aXe only tests up to WCAG AA. Not seeing anything in their docs for enabling AAA testing.

**Comment by ryanoglesby08** [comment]
Ahhh, ok then. I was thinking that may be the case. Alright, I guess there's nothing we can do then. Are there any issues or requests for AAA settings in aXe core? May be good to do a quick look?

**Comment by ryanoglesby08** [comment]
@marcod1419 @theetrain 

Quick question: is this a fix, feature, or breaking change?

While the API hasn't changed, the style has been visibly altered, which may be surprising to consumers...

**Pull request merged**